### PR TITLE
Re navbar side menu

### PR DESCRIPTION
**Pull request by MuhammadKhalilzadeh**
Navbar and side menu are fixed and redesigned, although there are some parts of it that needs to be completed as we move forward
Screenshot (81)

### CODE REVIEW

**Review by veyselboybay — APPROVED** [review]

**Comment by MuhammadKhalilzadeh** [comment]
I just changed the font for drop down menu on navbar and also managed to remove the animation effect with talk about for the menu toggle, but couldn't find a way to remove it for menu items too.

File: Client/src/Components/NavBar/index.css [hunk 1]
```diff
@@ -1 +1,29 @@
...
+.icon-button-toggle-pic {
+  width: 10px;
+  height: 5px;
+}
+
+#icon-button {
+  color: transparent;
+  -webkit-transition: none;
+  transition: none;
+}
```
Comment (Skorpios604) [inline Client/src/Components/NavBar/index.css:29]: Hey! If you want to get rid of the black outline that pops up when you click on the user dropdown menu you can use this here:
```CSS
outline: none; 
box-shadow: none;

Comment (Skorpios604) [reply Client/src/Components/NavBar/index.css:29]: You can also create a new CSS class like this to get rid of the greyish hue that pops up when you hover over the user dropdown. 
```CSS
#icon-button:hover {
  background-color: transparent; 
}

**Review by Skorpios604 — CHANGES_REQUESTED** [review]

File: Client/src/Components/NavBar/index.css (hunk 1 above)
Comment (MuhammadKhalilzadeh) [reply Client/src/Components/NavBar/index.css:32]: The first part worked, but the second part did not make a different and still effects on drop down menu do not disappear. By the way I think it's pretty much what we wanted on the UI.
Next, I'll be adding routing and navigation on this pr.

Comment (MuhammadKhalilzadeh) [reply Client/src/Components/NavBar/index.css:32]: I just added navigation for side menu using useNavigate and now pages render one the right side

**Review by veyselboybay — APPROVED** [review]
Overall looks good to me. There are 2 styling affect issues, we can come back for fix later. Moving forward.

**Review by ajhollid — APPROVED** [review]

**Pull request merged**